editor.planx.uk/src/pages/FlowEditor/utils.ts

```diff
@@ -17,4 +17,7 @@ export const formatLastEditMessage = (
 
   const name = `${actor.firstName} ${actor.lastName}`;
   return `Last edited ${formatLastEditDate(date)} by ${name}`;
-};
\ No newline at end of file
+};
+
+export const formatLastPublishMessage = (date: string, user: string): string =>
+  `Last published ${formatLastEditDate(date)} ago by ${user}`;
```

Comment:
This is just a small tidy up so that all of the helper methods that rely on `formatDistanceToNow()` are co-located in the same file